src/DistributedLock.ZooKeeper/DistributedLock.ZooKeeper.csproj

```diff
@@ -45,19 +45,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ZooKeeperNetEx" Version="3.4.12.4" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Nullable" Version="1.3.1">
+    <PackageReference Include="ZooKeeperNetEx" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
+    <PackageReference Include="Nullable" Condition="'$(TargetFramework)' != 'netstandard2.1'">
```

Comment:
Given `<TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>` on line 4 of this file, I think it was an oversight that this `Condition` didn't exist. Note that this change is reflected in the lock file diff:
https://github.com/madelson/DistributedLock/pull/186/commits/c87fb79e853c7b5ab38e980ec605d711bda38c23#diff-b74c05733e0ca95b4f1c079e5455270db5118c2ba952b1cbc46d7a1452a8fb18L175

Reply:
Does it break without this condition? I would have thought the package just noops in that case.